.github/workflows/pull_request.yaml

```diff
@@ -0,0 +1,92 @@
+name: Pull request
+
+on: pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+        fetch-depth: 0
+    - name: Build Apps DNS image
+      uses: docker/build-push-action@v2
+      with:
+        file: image/Dockerfile
+        tags: kubecf-apps-dns:latest
+        pull: true
+        push: false
+    - name: Build Acceptance tests image
+      uses: docker/build-push-action@v2
+      with:
+        file: tests/acceptance/image/Dockerfile
+        context: tests/acceptance/
+        tags: acceptance-tests:latest
+        pull: true
+        push: false
+    - uses: azure/setup-kubectl@v1
+    - uses: azure/setup-helm@v1
+    - name: Setup k3s
+      uses: SUSE/k3s-actions/create@v0.1.0
+    - name: Load the images into k3s
+      uses: SUSE/k3s-actions/images-import@v0.2.0
+      with:
+        images:
```

Comment:
Huh this doesn't need a `|`? I wonder how this reads as JSON. (That's two subkeys, with values of `latest`? Or a string with two lines?)